src/borg/testsuite/archiver/rdelete_cmd.py

```diff
@@ -1,25 +1,22 @@
 import os
 
-import pytest
-
 from ...constants import *  # NOQA
-from . import create_regular_file, cmd, RK_ENCRYPTION
+from . import create_regular_file, cmd, generate_archiver_tests, RK_ENCRYPTION
+
+pytest_generate_tests = lambda metafunc: generate_archiver_tests(metafunc, kinds="local,remote,binary")  # NOQA
 
 
-@pytest.mark.parametrize("archivers", ["archiver", "remote_archiver", "binary_archiver"])
```

Comment:
did you do a global search for other places you have forgotten to refactor to the new method?

Reply:
looks like this and test_rename was all, so this is done.